cluster/gce/config-default.sh

```diff
@@ -44,6 +44,8 @@ CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.244.0.0/16}"
 MINION_SCOPES=("storage-ro" "compute-rw" "https://www.googleapis.com/auth/monitoring" "https://www.googleapis.com/auth/logging.write")
 # Increase the sleep interval value if concerned about API rate limits. 3, in seconds, is the default.
 POLL_SLEEP_INTERVAL=3
+# TODO(roberthbailey): If SERVICE_CLUSTER_IP_RANGE is smaller than a /24 then
```

Comment:
We should enforce this.  can we parse this somewhere (maybe your create-certs func, or maybe earlier) and assert that the mask is <=24?